src/home-page/home-page.component.ts

```diff
@@ -0,0 +1,29 @@
+import {Component, OnInit} from '@angular/core';
+import {HttpClient} from '@angular/common/http';
+
+@Component({
+  selector: 'app-home-page',
+  templateUrl: './home-page.component.html',
+  styleUrls: ['./home-page.component.scss']
+})
+export class HomePageComponent implements OnInit {
+  public cityName: string;
+  public cities = [];
+
+  constructor(private http: HttpClient) {
+  }
+
+  ngOnInit() {
+  }
+
+  searchCity() {
+    this.http.get(`//api.openweathermap.org/data/2.5/weather?q=${this.cityName}&APPID=9255e12d14f3883315c070a9d1899c07`)
```

Comment:
You should move `//api.openweathermap.org/data/2.5/` to some constant field because I believe this part will be shared with other API calls.

Also you should export `9255e12d14f3883315c070a9d1899c07` to environments file in Angular

Reply:
I guess you can move both of them to environments file